src/storage-client/src/controller.proto

```diff
@@ -27,7 +27,8 @@ message ProtoCollectionMetadata {
 
 message ProtoDurableCollectionMetadata {
     // This message is persisted to disk. Changes must be backwards compatible.
-    reserved 1, 2, 5;
+    reserved 1, 2, 4, 5;
+    reserved "remap_shard";
     string data_shard = 3;
-    optional string remap_shard = 4;
+    // optional string remap_shard = 4;
```

Comment:
I think of this as documentation for what the reserved field ID/name correlation was, which is annoying to fish out of git because all reserved elements end up on the same line. certainly not required. @ParkMyCar thoughts?